docker/fix_classpaths.sh

```diff
@@ -16,7 +16,7 @@ sed -i 's/javax.json.jar/jakarta.json.jar/g' ${CTS_HOME}/jakartaeetck/bin/ts.jte
 sed -i 's/javax.json-api.jar/jakarta.json.jar/g' ${CTS_HOME}/jakartaeetck/bin/ts.jte
 sed -i 's/javax.mail.jar/jakarta.mail.jar/g' ${CTS_HOME}/jakartaeetck/bin/ts.jte
 sed -i 's/javax.management.j2ee-api.jar/jakarta.management.j2ee-api.jar/g' ${CTS_HOME}/jakartaeetck/bin/ts.jte
-sed -i 's/javax.persistence.jar/jakarta.persistence.jar/g' ${CTS_HOME}/jakartaeetck/bin/ts.jte
+sed -i 's/jakarta.persistence.jar/jakarta.persistence.jar/g' ${CTS_HOME}/jakartaeetck/bin/ts.jte
```

Comment:
this file replaces occurrences of javax.* in the ts.jte with jakarta.* . Hence will not require to change this.

Reply:
this comment has been addressed in the commit https://github.com/eclipse-ee4j/jakartaee-tck/pull/249/commits/166a1f1a1f11bbd091325ab83ab7fc5f1c11338d